src/views/bridge-details/bridge-details.view.tsx

```diff
@@ -134,6 +134,10 @@ const BridgeDetails: FC = () => {
           networkId,
         })
           .then((bridge) => {
+            if (bridge.destinationAddress !== connectedProvider.data.account) {
+              return navigate(routes.activity.path);
+            }
```

Comment:
I think this should be inside the `callIfMounted` block below. I guess we don't want to redirect the user when she has already left the page.